backend/src/main/java/de/neuefische/capstone/backend/ProjectCalculations.java

```diff
@@ -25,18 +25,18 @@ public Project calculateProgressForDonations(Project project) {
                     project.goal(),
                     project.location(),
                     project.donations(),
-                    project.volunteers(),
+                    project.participations(),
                     project.userId(),
                     project.image()
             );
         }
         return project;
     }
 
-    public Project calculateProgressForVolunteers(Project project) {
+    public Project calculateProgressForParticipations(Project project) {
         if(project.category() == Category.PARTICIPATION) {
-            int newVolunteers = project.volunteers().size();
```

Comment:
Why do you calculate with integer instead of BigDecimal?

Reply:
Because at this point I'm only checking how many people have volunteered. And since half people are not existing in this world, I chose integer 😃